uk.co.compucorp.civicrm.tasksassignments/api/v3/Task.php

```diff
@@ -611,8 +622,8 @@ function civicrm_api3_task_getstatuses($params) {
     return null;
 }
 
-function _civicrm_api3_task_gettypesbycomponent($component, $sequential = 1) {
-    
+function _civicrm_api3_task_gettypesbycomponent($component, $params = []) {
+
```

Comment:
@omarabuhussein absolutely inconceivable, this space has to stay! :-) 

I wasn't sure whether to add a space since the other functions are a mix. Good to know for the future.